src/operator/switch.ts

```diff
@@ -1,11 +1,10 @@
 import {Operator} from '../Operator';
-import {Observable} from '../Observable';
 import {Subscriber} from '../Subscriber';
 import {Subscription} from '../Subscription';
 import {OuterSubscriber} from '../OuterSubscriber';
 import {subscribeToResult} from '../util/subscribeToResult';
 
-export function _switch<T>(): Observable<T> {
+export function _switch<T>(): T {
```

Comment:
Yes, now can sense that. Let's continue discussion on https://github.com/ReactiveX/RxJS/pull/1077#discussion_r47714337.